src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+public enum Rank {
+    FOURTH(3, 5000),
+    THIRD(4, 50000),
+    SECOND(5, 1500000),
+    FIRST(6, 2000000000);
```

Comment:
enum 활용 👍 
```suggestion
    FOURTH(3, 5_000),
    THIRD(4, 50_000),
    SECOND(5, 1_500_000),
    FIRST(6, 2_000_000_000);
```
가독성을 위해 이처럼 표현이 가능해요

Reply:
언더바로 표현이 가능하다는 걸 처음 알았습니다~!
감사합니다~!! 👍

Reply:
언더바로 표현이 가능하다는 걸 처음 알았습니다~!
감사합니다~~! 👍